### ISSUE TITLE
Update VAOS dashboard

### ISSUE THREAD

**Issue by markgreenburg** [department-of-veterans-affairs/va.gov-team#2143]
- Remove 'get started' sub-header
- Update help content under 'manage appointments' heading to 'View or cancel appointments'

**Comment by jenniemc** [comment]
@markgreenburg, I don't see the sub header 'manage appointments' on the dashboard. However I see 'View your appointments'. Would you mean to change 'View your appointments' to 'View or cancel appointments'?

**Comment by markgreenburg** [comment]
@jenniemc yep! I think the content update fell behind staging but you're correct.
